applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -404,16 +404,23 @@ int le_audio_volume_mute(void)
 	return hw_codec_volume_mute();
 }
 
-int le_audio_play(void)
+int le_audio_play_pause(void)
 {
-	playing_state = true;
-	return 0;
-}
+	int ret;
 
-int le_audio_pause(void)
-{
-	playing_state = false;
-	return bt_audio_broadcast_sink_stop(broadcast_sink);
+	if (playing_state) {
+		playing_state = false;
+
+		ret = bt_audio_broadcast_sink_stop(broadcast_sink);
+		if (ret) {
+			LOG_ERR("Failed to stop broadcast sink: %d", ret);
+			return ret;
+		}
+	} else {
+		playing_state = true;
```

Comment:
I may see this later, but I don't immediately get how setting this to true starts the playback.

Reply:
In case you didn't see it it's in syncable_cb. This is because we enter this callback every time we receive BIG info. This is sent periodically by the gateway as long as it is broadcasting